Sources/Keystone/WordPress.h

```diff
@@ -0,0 +1,82 @@
+#import <Foundation/Foundation.h>
+
+//! Project version number for WordPress.
+FOUNDATION_EXPORT double WordPressVersionNumber;
+
+//! Project version string for WordPress.
+FOUNDATION_EXPORT const unsigned char WordPressVersionString[];
+
+// In this header, you should import all the public headers of your framework using statements like #import <WordPress/PublicHeader.h>
+
+#import <WordPress/ActivityLogViewController.h>
+#import <WordPress/AccountService.h>
+
+#import <WordPress/Blog.h>
+#import <WordPress/BlogService.h>
+#import <WordPress/BlogSyncFacade.h>
+#import <WordPress/BlogDetailsViewController.h>
+
+#import <WordPress/CommentService.h>
+#import <WordPress/CommentsViewController.h>
+#import <WordPress/Constants.h>
+
+#import <WordPress/LocalCoreDataService.h>
+
+#import <WordPress/Media.h>
+#import <WordPress/MediaService.h>
+#import <WordPress/MenuItem.h>
+#import <WordPress/MenuItemsViewController.h>
+#import <WordPress/MenusService.h>
+#import <WordPress/MenusViewController.h>
+#import <WordPress/Media+Extensions.h>
+
+#import <WordPress/NSObject+Helpers.h>
+
+#import <WordPress/PageSettingsViewController.h>
+#import <WordPress/PostContentProvider.h>
+#import <WordPress/PostCategory.h>
+#import <WordPress/PostCategoryService.h>
+#import <WordPress/PostContentProvider.h>
+#import <WordPress/PostHelper.h>
+#import <WordPress/PostMetaButton.h>
+#import <WordPress/PostService.h>
+#import <WordPress/PostServiceOptions.h>
+#import <WordPress/PostSettingsViewController.h>
+#import <WordPress/PostTag.h>
+#import <WordPress/PostTagService.h>
+
+#import <WordPress/ReaderGapMarker.h>
+#import <WordPress/ReaderPost.h>
+#import <WordPress/ReaderPostService.h>
+#import <WordPress/ReaderSiteService.h>
+#import <WordPress/ReaderSiteService_Internal.h>
+#import <WordPress/ReaderTopicService.h>
+
+#import <WordPress/SettingsSelectionViewController.h>
+#import <WordPress/SettingsMultiTextViewController.h>
+#import <WordPress/SettingTableViewCell.h>
+#import <WordPress/SettingsTextViewController.h>
+#import <WordPress/SharingViewController.h>
+#import <WordPress/SiteSettingsViewController.h>
+#import <WordPress/SourcePostAttribution.h>
+#import <WordPress/StatsViewController.h>
+#import <WordPress/SuggestionsTableView.h>
+#import <WordPress/SuggestionsTableViewCell.h>
+
+#import <WordPress/Theme.h>
+#import <WordPress/ThemeService.h>
+
+#import <WordPress/UIAlertControllerProxy.h>
+#import <WordPress/UIApplication+Helpers.h>
+#import <WordPress/UIView+Subviews.h>
+
+#import <WordPress/WPAccount.h>
+#import <WordPress/WPActivityDefaults.h>
+#import <WordPress/WPAppAnalytics.h>
+#import <WordPress/WPUploadStatusButton.h>
+#import <WordPress/WPError.h>
+#import <WordPress/WPTableViewHandler.h>
+#import <WordPress/WPTabBarController.h>
+#import <WordPress/WPLogger.h>
+
+FOUNDATION_EXTERN void SetCocoaLumberjackObjCLogLevel(NSUInteger ddLogLevelRawValue);
```

Comment:
Why did you decide to rename Keystone to WordPress?

Reply:
It's a temporary change to avoid changing IB files and Core Data schemas.